docs/sql_statements/data_definition_statements.md

```diff
@@ -214,7 +214,7 @@ Query OK, 1 row affected (0.00 sec)
 CREATE [UNIQUE | FULLTEXT | SPATIAL] INDEX index_name
     ON tbl_name (key_part,...)
     [index_option]
-    [algorithm_option | lock_option] ...
+    [algorithm_option | lock_option | algorithm_option lock_option | lock_option algorithm_option]
```

Comment:
https://dev.mysql.com/doc/refman/8.0/en/drop-index.html

Reply:
> Be consistent with the documentation https://dev.mysql.com/doc/refman/8.0/en/create-index.html
There is a mistake in the official 8.0 description,  here we corrected it.